keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/UserResource.java

```diff
@@ -1,23 +1,12 @@
 package io.cloudtrust.keycloak.services.resource.api.admin;
 
-import org.keycloak.events.admin.ResourceType;
 import org.keycloak.models.KeycloakSession;
 import org.keycloak.models.UserModel;
 import org.keycloak.services.resources.admin.AdminEventBuilder;
 import org.keycloak.services.resources.admin.permissions.AdminPermissionEvaluator;
 
 public class UserResource extends org.keycloak.services.resources.admin.UserResource {
-
-    private AdminPermissionEvaluator auth;
-
-    private AdminEventBuilder adminEvent;
-    private UserModel user;
-
     public UserResource(KeycloakSession session, UserModel user, AdminPermissionEvaluator auth, AdminEventBuilder adminEvent) {
         super(session.getContext().getRealm(), user, auth, adminEvent);
```

Comment:
Why do we have this class ? It just overrides UserResource with bringing any new feature... May be we should remove it?

Reply:
You meant "without bringing", right? Indeed, we should remove it.

Reply:
Removed